ResourceStatusSystem/Command/DowntimeCommand.py

```diff
@@ -106,9 +137,18 @@ def _prepareCommand( self ):
       if not seHost:
         return S_ERROR( 'No seHost for %s' % elementName )
       elementName = seHost
+      
+    elif elementType == 'FTS':
+      gocdbServiceType = elementType
+      try:
+        #WARNING: this method presupposes that the server is an FTS3 type
```

Comment:
Honestly I'm not sure about all use cases. However I'm more sure about the fact that GOCDB doesn't have FTS3 as a type and that, at this stage, we are dealing only with FTS3. So I'll put "elementType == 'FTS'  or elementType == 'FTS3'". Unless you prefer otherwise.